include/fastdds/dds/publisher/DataWriter.hpp

```diff
@@ -174,6 +193,22 @@ class DataWriter : public DomainEntity
     RTPS_DllAPI InstanceHandle_t register_instance(
             void* instance);
 
+    /** NOT YET IMPLEMENTED
+     * @brief This operation performs the same function as register_instance and can be used instead of
+     * @ref register_instance in the cases where the application desires to specify the value for the
+     * @ref source_timestamp. The @ref source_timestamp potentially affects the relative order in which readers observe
+     * events from multiple writers. See the QoS policy @ref DESTINATION_ORDER. This operation may block and return
+     * RETCODE_TIMEOUT under the same circumstances described for the @ref write operation. This operation may return
```

Comment:
```suggestion
     * RETCODE_TIMEOUT under the same circumstances described for the @ref register_instance operation. This operation may return
```

Reply:
As per the [DDS Standard 1.4](https://www.omg.org/spec/DDS/1.4/PDF) p.51 these error codes are defined with respect to the write operation.